azure-pipelines.yml

```diff
@@ -179,9 +179,50 @@ stages:
                testRunTitle: Functional Test Results
              displayName: 'Publish Functional Test Results'
 
-      - job: AccessibilityTests
+      - job: E2ETests
         dependsOn: FunctionalTests
         pool: $(WindowPool)
+        displayName: "End-To-End Tests"
+        steps:
+           - checkout: self
+           - script: 
```

Comment:
Do we need this echo?

Reply:
Echo can be removed from ATs and ETs. it was just added to validate the URL as present in FT.